src/views/GraphView/index.tsx

```diff
@@ -107,7 +107,8 @@ const GraphView = ({ history }) => {
     <div className={
       `data-view ${detailPanelIsOpen
         ? 'data-view--squished'
-        : ''}`}
+        : ''}`
+}
```

Comment:
more autofix werd spacing

Reply:
fixed in [edb87f9](https://github.com/bcgsc/pori_graphkb_client/pull/51/commits/edb87f9b15f38bb438ce748a51586105f00b3773)